src/fission.F90

```diff
@@ -26,7 +25,7 @@ function nu_total(nuc, E) result(nu)
     real(8) :: c  ! polynomial coefficient
 
     if (nuc % nu_t_type == NU_NONE) then
-      call fatal_error("No neutron emission data for table: " // nuc % name)
+      nu = ERROR_REAL
```

Comment:
Yes, in order to be `pure`, a procedure can't call `fatal_error` since that in turn calls a number of non-pure procedures. I do think we ought to be more methodical about error codes.